Library/AppEnvironmentTests+OAuthInKeychain.swift

```diff
@@ -0,0 +1,422 @@
+@testable import KsApi
+@testable import Library
+import XCTest
+
+final class AppEnvironmentTests_OAuthInKeychain: XCTestCase {
+  override func setUp() {
+    AppEnvironment.resetStackForUnitTests()
+    XCTAssertNoThrow(try Keychain.deleteAllPasswords())
+  }
+
+  func setFeatureUseKeychainEnabled(_ setting: Bool) {
+    let mockConfigClient = MockRemoteConfigClient()
+    mockConfigClient.features = [
+      RemoteConfigFeature.useKeychainForOAuthToken.rawValue: setting
+    ]
+
+    /* Awkward bit of condition here - our feature flag values implicitly come from
+     AppEnvironment.current, which in some cases, is the PREVIOUS environment in the stack.
+     If you're pushing/saving a new stack, that can be confusing. */
+    AppEnvironment.updateRemoteConfigClient(mockConfigClient)
```

Comment:
This was an interesting mess I got myself into - what does it mean to test a feature flag from within `AppEnvironment`, which is _where we store the feature flags_?